packages/edit-post/src/hooks/components/media-upload/index.js

```diff
@@ -67,7 +67,6 @@ const getAttachmentsCollection = ( ids ) => {
 	return wp.media.query( {
 		order: 'ASC',
 		orderby: 'post__in',
-		per_page: -1,
```

Comment:
@youknowriad If we remove `per_page=-1` here, then we'll only get up to 10 items in the response. This seems like it will result in buggy behavior.

Reply:
In the interest of time, I'd be fine to change this to `per_page=100` if we properly resolve all data in 4.2

Reply:
This request explicitly fetch `ids` so you're right it can probably happen but I'd say very unlikely if we use a big number. Let's go for 100 for now.

Also, worth noting that I searched the history and originally it was written `perPage: -1` and no issue was raised.